#wheel_width: The width of the wheels
#wheel_radius: The radius of the wheels
#wheel_base: The distance between the centers wheels
#chassis_length: The length of the main chassis link
#chassis_thickness: The thickness of the plate that forms the chassis

wheel_radius: 0.2
wheel_base: 1.2
frequency: 100
#wheel_radius: 1.0
#wheel_base: 1.0
#frequency: 1
odom_frame_id: 'odom'
body_frame_id: 'base_link'
left_wheel_joint: 'left_wheel_axle'
right_wheel_joint: 'right_wheel_axle'